vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -101,7 +101,7 @@
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <ForcedIncludeFiles>pch.h</ForcedIncludeFiles>
-      <WarningLevel>Level4</WarningLevel>
+      <WarningLevel>Level3</WarningLevel>
```

Comment:
Why is this needed? Seems like this will disable a bunch of warnings

Reply:
Not needed, I'll change that back.